apps/portal/src/components/color-palette.tsx

```diff
@@ -0,0 +1,77 @@
+import * as styles from "@harnessio/core-design-system/styles-esm";
+import { Tooltip } from "@harnessio/ui/components";
+
+interface Color {
+  name?: string;
+  $value?: string;
+  [key: string]: unknown;
+}
+
+export function ColorPalette() {
+  const shades = Object.keys(styles.colors.blue);
+  const colors = Object.keys(styles.colors).filter(
+    (color) => color !== "pure",
+  ); /** Excluding "pure" colors intentionally */
+
+  return (
+    <div className="grid rounded-lg border border-borders-4 bg-background-4 px-6 pt-1 pb-6 mt-4 shadow-md h-fit w-[800px]">
```

Comment:
Setting a hard width on this will mean we struggle to adapt the app to be repsonsive. Grids are great at adjusting. Why don't we lean on grid and set the aspect ratio of the cells to keep as square?

Reply:
Had a look and 800 is wider than the body of the app, even fully expanded

Reply:
Makes sense, removed the hardcoded width.